buildspec.yml

```diff
@@ -32,39 +30,24 @@ phases:
       # run unit tests
       - pytest test/unit
 
-      # Create pip archive
+      # Create pip archive in dist/
+      # tar_name will be something like: sagemaker_mxnet_container-3.0.0.tar.gz
+      - python3 setup.py sdist
```

Comment:
does python3 vs python2.7 affect this?

Reply:
not sure, but we've always done this. a better approach might be to use bdist_wheel like Chainer/PyTorch do, which generates a binary that's explicitly for both Python versions, but I think that's out of scope here.